.yamllint

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
precommit checks were failing on yamllint without this config file. I've copied this from the cdk handler repo.